src/Common/src/Interop/Windows/user32/Interop.GetWindowText.cs

```diff
@@ -10,7 +10,7 @@ internal partial class Interop
 {
     internal partial class User32
     {
-        [DllImport(Libraries.User32)]
+        [DllImport(Libraries.User32, EntryPoint = "GetWindowTextW")]
```

Comment:
Does this work correctly on 32-bit? I'd be concerned that it's targeting the W overload but would use the default Ansi rather than Unicode marshaling.